src/components/Auth/OneAccountPerEmailCard/OneAccountPerEmailDialog.tsx

```diff
@@ -0,0 +1,99 @@
+import {
+  Dialog,
+  DialogActions,
+  DialogButton,
+  DialogContent,
+  DialogTitle,
+} from '@rmwc/dialog';
+import { Radio } from '@rmwc/radio';
+import { Typography } from '@rmwc/typography';
+import React from 'react';
+import { MapDispatchToPropsFunction, connect } from 'react-redux';
+
+import { createStructuredSelector } from '../../../store';
+import { setAllowDuplicateEmailsRequest } from '../../../store/auth/actions';
+import { getAllowDuplicateEmails } from '../../../store/auth/selectors';
+import styles from './OneAccountPerEmailDialog.module.scss';
+
+interface OneAccountPerEmailDialogProps {
+  onClose: () => void;
+}
+
+type Props = PropsFromDispatch & PropsFromStore & OneAccountPerEmailDialogProps;
+
+export const OneAccountPerEmailDialog: React.FC<Props> = ({
+  allowDuplicateEmails,
+  onClose,
+  setAllowDuplicateEmails,
+}) => {
+  const [value, setValue] = React.useState(allowDuplicateEmails);
+
+  return (
+    <Dialog renderToPortal open onClose={onClose}>
+      <DialogTitle>Account email address settings</DialogTitle>
+      <form
+        onSubmit={(e: React.FormEvent<unknown>) => {
+          e.preventDefault();
+          setAllowDuplicateEmails(value);
+          onClose();
+        }}
+      >
+        <DialogContent>
+          <Typography use="body2" className={styles.description}>
+            This setting either prevents or allows users to create multiple
+            accounts using the same email address if they sign in to your app
+            using different authentication providers.
+          </Typography>
+          <Radio
+            value="false"
+            checked={!value}
+            onChange={() => setValue(false)}
```

Comment:
Wouldn't this also trigger when it's unchecked? Also, do we need `onChange` + `onClick` on both radios? Wouldn't `onChange` for the first one just work (with no `onClick` at all, no `onChange` on the second radio)?

Reply:
onChange actually works, but I spent an hour trying to make tests work with fireEvent change, and it just never called the callback for some reason, so I added a click handler which works just fine.

OnChange is only called for check, and not for uncheck.